GTG/core/meson.build

```diff
@@ -1,3 +1,16 @@
+infodir = python3.get_install_dir()
+version = run_command('git', 'describe', '--dirty').stdout().strip()
```

Comment:
I think it should handle the case when `git` isn't available or fails (such as when the source tarball or the download-as-zip on github is being used). (Actually didn't test what happens)
Maybe fall back to the version as [defined in the main `meson.build` file](https://github.com/getting-things-gnome/gtg/blob/d7bbcd5b2a9d7d54b2b28278e5bbf6fd264b302b/meson.build#L2)? (Seems `meson.project_version()` would return that version).

Reply:
It's a bit longish now, but it works both when git is not present and when we are not building from a repo